dev-manuals/test-plans/overworld-frontend/plans/npc-load-dialogue.md

```diff
@@ -0,0 +1,37 @@
+# Name: NPC load dialogue
```

Comment:
```suggestion
# NPC load dialogue
```
Also, what about using the usecase number in the title as well? That'll be better for the `ReadTheDocs` integration:
```suggestion
# NPC load dialogue (`U.NPC.2`)
```